cfme/networks/balancer.py

```diff
@@ -0,0 +1,84 @@
+from utils import version
+from utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
+from navmazing import NavigateToSibling, NavigateToAttribute
+from utils.appliance import Navigatable
+from utils.update import Updateable
+from cfme.common import Taggable, SummaryMixin
+from cfme.networks.views import BalancerView
+from cfme.networks.views import BalancerDetailsView
+
+
+class BalancerCollection(Navigatable):
+    ''' Collection object for Balancer object
+    '''
```

Comment:
**REQUIRED**
Here and below
http://cfme-tests.readthedocs.io/guides/dev_guide.html#code-style
> According to PEP 8, triple-quoted docstrings use double quotes. To help differentiate docstrings from normal multi-line strings, consider using single-quotes in the latter case: